torchtune/utils/_checkpointing/_checkpointer_utils.py

```diff
@@ -11,14 +11,17 @@
 import torch
 import torch.nn as nn
 import torch.optim as optim
+from safetensors import safe_open
```

Comment:
Can you add this to requirements.txt?

Reply:
Yeah, thank you for your suggestion, I add it in the latest version.